libs/api-test-environment/src/main/java/de/cyface/apitestutils/fixture/user/TestUser.java

```diff
@@ -115,8 +115,11 @@ protected List<String> getRoles() {
      */
     protected String getHashedPassword() {
         final var salt = "cyface-salt";
-        final var hasher = new Hasher(HashingStrategy.load(), salt.getBytes(StandardCharsets.UTF_8));
-        return hasher.hash(password);
+        //noinspection SpellCheckingInspection
+        return HashingStrategy.load().hash("pbkdf2",
+                null,
+                Base64.getMimeEncoder().encodeToString(salt.getBytes(StandardCharsets.UTF_8)),
+                password);
```

Comment:
This was necessary to make the libs/api-test-environment not dependent of libs/api